vcd/resource_vcd_vapp_network_test.go

```diff
@@ -61,11 +62,15 @@ func TestAccVcdVappNetwork_Isolated(t *testing.T) {
 		"EdgeGateway":                 testConfig.Networking.EdgeGateway,
 		"NetworkName":                 "TestAccVcdVAppNet",
 		"NetworkName2":                "TestAccVcdVAppNet2",
-		"orgNetwork":                  "",
-		"orgNetworkForUpdate":         "",
+		"OrgNetworkKey":               " ",
```

Comment:
Could we not use `delete(map, key)` (or not set it at all) instead of a hack with `" "` blank space

Reply:
we can't provide values even empty ones, as template fill will fail, and check in the test will fail even if the template could be passed. A moment doesn't have better handling dynamic templating.

Reply:
One can check if a key exists in a map using templates as well. But let it be as this is a very rare occasion that this hack is used.

Reply:
then we automatically have to replace in template not existing keys with empty spaces, which will create other unexpected behaviors. This workaround comes to play when you use dynamic templating: {{.key}} {{.equalsChar}} {{.value}}. So when have generic run tests and resources with a different set of configurations.

Reply:
add comment there

Reply:
> requires that values isn't empty

->

> requires that values aren't empty

Can you also describe **why** you need to pass these parameters as supposedly empty?

Reply:
rephrased comment